### PR TITLE
docs(tutorials): fixed various tutorials MTA-1462

### PR DESCRIPTION
**Pull request by RoRoJ**
Reviewed/updated 2 tutorials due review
Various other small fixes

### CODE REVIEW

File: tutorials/erpnext/index.mdx [hunk 1]
```diff
@@ -30,15 +30,21 @@ categories:
...
 ## ERPNext - Overview
 
 [ERPNext](https://www.erpnext.com/) is a free and open-source ERP software designed to be used by manufacturers, distributors, and service companies. It includes a full range of modules, including accounting, sales, CRM, purchasing, warehouse management, and inventory. Moreover, specialized modules are available tailored to schools, healthcare, agriculture, and non-profit sectors. It uses a MariaDB database to store its data, is written in Python, and based on the [Frappé Framework](https://frappeframework.com/). The application is released under the GNU GPLv3 license, and its source code is hosted on [GitHub](https://github.com/frappe/erpnext).
 
-In this tutorial, you will learn how to install and configure ERPNext on Ubuntu 18.04 (Bionic Beaver) on a Scaleway Dedibox dedicated server.
+In this tutorial, you will learn how to install and configure ERPNext 12 on Ubuntu 18.04 (Bionic Beaver) on a Scaleway Dedibox dedicated server.
+
+<Message type="tip">
+
+  See [our updated tutorial](/tutorials/erpnext-13) for installing ERPNext 13 on Ubuntu 20.04 Focal Fossa
```
Comment (ldecarvalho-doc) [inline tutorials/erpnext/index.mdx:45]:
```suggestion
  See [our updated tutorial](/tutorials/erpnext-13/) for installing ERPNext 13 on Ubuntu 20.04 Focal Fossa
```

File: tutorials/erpnext/index.mdx [hunk 2]
```diff
@@ -196,19 +214,19 @@ All following steps will be executed from the newly created user account and by
   sudo bash nodesource_setup.sh
   ```
 
-5. Now install NodeSource usin `apt`:
+5. Install NodeSource usin `apt`:
```
Comment (ldecarvalho-doc) [inline tutorials/erpnext/index.mdx:217]:
```suggestion
5. Install NodeSource using `apt`:
```

File: tutorials/erpnext/index.mdx [hunk 3]
```diff
@@ -220,19 +238,19 @@ All following steps will be executed from the newly created user account and by
...
 
-10. Once the files are in place, make them executable:
+10. Make the files executable:
 
   ```
   sudo chmod a+x /usr/bin/wkhtmlto*
   ```
 
-11. Then return back into the `/home` directory of the user:
+11. Return back into the `/home` directory of the user:
```
Comment (ldecarvalho-doc) [inline tutorials/erpnext/index.mdx:253]:
```suggestion
11. Return to the `/home` directory of the user:
```

File: tutorials/erpnext/index.mdx [hunk 4]
```diff
@@ -356,28 +374,28 @@ Our test environment has proven that our ERPNext installation is working, but so
...
 
     - **Nginx:** Two configuration files: `/etc/nginx/nginx.conf`, and `/etc/nginx/conf.d/frappe-bench.conf`
     - **Fail2Ban:** One proxy jail (`/etc/fail2ban/jail.d/nginx-proxy.conf`), and one filter configuration (`/etc/fail2ban/filter.d/nginx-proxy.conf`)
 
   The default values configured in these files are suitable for an essential start and the goals of this tutorial. However, depending on your workload, you may tweak them further.
 
-2. To increase security, we will use TLS to encrypt the connection between clients and the server. For this purpose, we install [`certbot`](https://certbot.eff.org) on the machine, a tool that can manage the request and renewal of Let's Encrypt certificates and the automatic configuration of Nginx. Add the certbot packet archive to the `apt` package manage:
+2. Use TLS to encrypt the connection between clients and the server for increased security. For this purpose, install [`certbot`](https://certbot.eff.org) on the machine, a tool that can manage the request and renewal of Let's Encrypt certificates and the automatic configuration of Nginx. Add the certbot packet archive to the `apt` package manage:
```
Comment (ldecarvalho-doc) [inline tutorials/erpnext/index.mdx:390]:
```suggestion
2. Use TLS to encrypt the connection between clients and the server for increased security. For this purpose, install [`certbot`](https://certbot.eff.org) on the machine. `certbot` is a tool that can manage the request and renewal of Let's Encrypt certificates and the automatic configuration of Nginx. Add the certbot packet archive to the `apt` package manage:
```

File: tutorials/erpnext/index.mdx [hunk 5]
```diff
@@ -356,28 +374,28 @@ Our test environment has proven that our ERPNext installation is working, but so
...
 
   ```
   sudo add-apt-repository universe
   sudo add-apt-repository ppa:certbot/certbot
   sudo apt update
   ```
 
-2. Then install certbot using `apt`:
+2. Install certbot using `apt`:
```
Comment (ldecarvalho-doc) [inline tutorials/erpnext/index.mdx:398]:
```suggestion
2. Install `certbot` using `apt`:
```

File: tutorials/odoo/index.mdx [hunk 6]
```diff
@@ -1,64 +1,66 @@
...
 tags: odoo, erp, cms, business management
 excerpt: |
   Odoo is a business management software, including different modules such as CRM, billing, e-commerce, warehouse, website builder, inventory management, and accounting. The software is distributed in an open-code model, providing a free Community Edition, released under the GNU LGPLv3 software license. The source code of the Community Edition is available on the companies GitHub. Besides the free and open-source Community Edition, a proprietary "Enterprise" version provides additional features and services. 
   The modular design of Odoo allows developers to create additional modules or apps and distribute them on the Odoo marketplace, which provides more than 20.000 modules. This fact makes Odoo a versatile and customizable solution that can be adapted to many usage scenarios. 
-  This tutorial will show you how to deploy Odoo 14 on a Scaleway Dedibox dedicated server, running on Ubuntu Focal Fossa (20.04 LTS).
+  This tutorial will show you how to deploy Odoo 14 on a Scaleway Instance or Scaleway Dedibox dedicated server, running on Ubuntu Focal Fossa (20.04 LTS).
```
Comment (ldecarvalho-doc) [inline tutorials/odoo/index.mdx:9]:
```suggestion
  This tutorial shows you how to deploy Odoo 14 on a Scaleway Instance or Scaleway Dedibox dedicated server, running on Ubuntu Focal Fossa (20.04 LTS).
```

File: tutorials/odoo/index.mdx [hunk 7]
```diff
@@ -1,64 +1,66 @@
...
+This tutorial will show you how to deploy Odoo 14 on a Scaleway machine running Ubuntu Focal Fossa (20.04 LTS). This could be a [Dedibox](https://www.scaleway.com/en/dedibox/), an [Instance](https://www.scaleway.com/en/virtual-instances/) or a [Bare Metal server](https://www.scaleway.com/en/bare-metal-servers/).
 
 
 <Message type="requirement">
 
 - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
 - You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
-- You have [created an Instance](/compute/instances/how-to/create-an-instance) that runs Ubuntu Bionic Beaver
+- You have one of the following remote machines running Ubuntu Focal Fossa (20.04 LTS):
+  - An [Instance](/compute/instances/how-to/create-an-instance)
```
Comment (ldecarvalho-doc) [inline tutorials/odoo/index.mdx:50]:
```suggestion
  - An [Instance](/compute/instances/how-to/create-an-instance/)
```

File: tutorials/odoo/index.mdx [hunk 8]
```diff
@@ -241,19 +256,19 @@ By default, the Odoo web server is using a plain HTTP connection to transport da
...
 
-3. Activate the site by creating a symlink in the `sites-enabled` directory:
+4. Activate the site by creating a symlink in the `sites-enabled` directory:
 
   ```
   ln -s /etc/nginx/sites-available/odoo.example.com /etc/nginx/sites-enabled/
   ```
 
-4. Run `certbot` to request a new certificate and configure Nginx. Follow the [instructions](/tutorials/configure-nginx-letsencrypt/#installing-certbot-and-obtaining-a-certificate) given during the certificate request:
+5. Run `certbot` to request a new certificate and configure Nginx. Follow the [instructions](/tutorials/configure-nginx-letsencrypt/#installing-certbot-and-obtaining-a-certificate) given during the certificate request:
```
Comment (ldecarvalho-doc) [inline tutorials/odoo/index.mdx:265]:
```suggestion
5. Run `certbot` to request a new certificate and configure Nginx. Follow the [instructions](/tutorials/configure-nginx-letsencrypt/#installing-certbot-and-obtaining-a-certificate/) given during the certificate request:
```

File: tutorials/erpnext/index.mdx [hunk 9]
```diff
@@ -1,6 +1,6 @@
 ---
-title: How to deploy ERPNext on Ubuntu 18.04 LTS (Bionic Beaver)
-description: Discover how to deploy the ERPNext application on a Scaleway Dedibox dedicated server
+title: How to deploy ERPNext 12 on Ubuntu 18.04 LTS (Bionic Beaver)
+description: Discover how to deploy the ERPNext 12 application on a Scaleway Dedibox dedicated server
 tags: erpnext erp odoo
 excerpt: |
   ERPNext is a free and open-source ERP software desigend to be used by manufacteres, distribuors and service companies. It includes a full range of modules including accounting, sales, CRM, purchasing, warehouse management and inventory. Moreover, specialised modules are available tailored to the needs of schools, healthcare, agriculture and non-proift sectors. It uses a MariaDB database to store its dat, is written in Python and based on the Frappé Framework. The application is released under the GNU GPLv3 license and its sourcecode is hosted on GitHub.
```
Comment (nedaehm) [inline tutorials/erpnext/index.mdx:6]:
```suggestion
  ERPNext is a free and open-source ERP software designed to be used by manufacturers, distributors and service companies. It includes a full range of modules including accounting, sales, CRM, purchasing, warehouse management and inventory. Moreover, specialized modules are available tailored to the needs of schools, healthcare, agriculture and non-profit sectors. It uses a MariaDB database to store its dat, is written in Python and based on the Frappé Framework. The application is released under the GNU GPLv3 license and its source code is hosted on GitHub.
```

File: tutorials/erpnext/index.mdx [hunk 10]
```diff
@@ -79,8 +90,15 @@ Start by configuring the system's keyboard mapping for the console as well as th
   LANG=en_US.UTF-8
   ```
 
+5. Configure the locale of the system to `en_US.utf8`:
+
+  ```
+  localectl set-keymap us && localectl set-locale LANG=en_US.utf8
+  ```
+
   Save the file, exit the text editor, and reboot your instance using the `reboot` command.
```
Comment (nedaehm) [inline tutorials/erpnext/index.mdx:99]:
```suggestion
  Save the file, exit the text editor, and reboot your Instance using the `reboot` command.
```

File: tutorials/erpnext/index.mdx [hunk 11]
```diff
@@ -272,13 +290,13 @@ As the ERPNext stack's major prerequisites are installed, go on by installing th
 
 ### Installing the Frappe Framework
 
-1. During the Frappe Framework installation, you may exceed Ubuntu’s file limit, increase it, and use the following command:
+1. Increase Ubuntu’s file limit, which may be necessary for sucessfully installing the Frappe Framework. Use the following command:
 
   ```
```
Comment (nedaehm) [inline tutorials/erpnext/index.mdx:295]:
```suggestion
  `echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p`
```

File: tutorials/erpnext/index.mdx [hunk 12]
```diff
@@ -272,13 +290,13 @@ As the ERPNext stack's major prerequisites are installed, go on by installing th
...
   echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
```
Comment (nedaehm) [inline tutorials/erpnext/index.mdx:296]:
```suggestion
```

File: tutorials/erpnext/index.mdx [hunk 13]
```diff
@@ -272,13 +290,13 @@ As the ERPNext stack's major prerequisites are installed, go on by installing th
...
   ```
```
Comment (nedaehm) [inline tutorials/erpnext/index.mdx:297]:
```suggestion
```

File: tutorials/erpnext/index.mdx [hunk 14]
```diff
@@ -356,28 +374,28 @@ Our test environment has proven that our ERPNext installation is working, but so
 - **Nginx**, a versatile web server that will be used as a proxy to redirect all requests on the standard HTTP (Port 80) and HTTPS (Port 443) to the ERPNext application listening on port 8000.
 - **Supervisor**, a service that ensures all required processes of ERPNext are constantly up and running. In case of any failure, it will restart them.
 
-1. The following command must be executed from the `frappe-bench` directory of your user. Run `bench` with these parameters and replace `timmy` with the name of your user owning the production environment:
+1. Executed the following command from the `frappe-bench` directory of your user. Run `bench` with these parameters and replace `timmy` with the name of your user owning the production environment:
```
Comment (nedaehm) [inline tutorials/erpnext/index.mdx:377]:
```suggestion
1. Execute the following command from the `frappe-bench` directory of your user. Run `bench` with these parameters and replace `timmy` with the name of your user owning the production environment:
```

File: tutorials/migrate-mysql-databases-postgresql-pgloader/index.mdx [hunk 15]
```diff
@@ -29,7 +29,8 @@ In this tutorial you will learn how to migrate from an existing remote MySQL dat
 
 <Message type="requirement">
 
-- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)
 - You have a [Database Instance](https://www.scaleway.com/en/database/)
 - You have a compute instance running on Ubuntu Bionic Beaver (18.04).
```
Comment (nedaehm) [inline tutorials/migrate-mysql-databases-postgresql-pgloader/index.mdx:35]:
```suggestion
- You have a Compute Instance running on Ubuntu Bionic Beaver (18.04).
```

File: tutorials/odoo/index.mdx [hunk 16]
```diff
@@ -1,83 +1,91 @@
 ---
 layout: docs
-title: Deploying Odoo ERP on a Scaleway Dedibox
+title: Deploying Odoo ERP on Ubuntu 20.04 LTS (Focal Fossa)
 description: Learn how to deploy Odoo, a versatile business management software on Ubuntu 20.04 LTS (Focal Fossa)
 tags: odoo, erp, cms, business management
 excerpt: |
   Odoo is a business management software, including different modules such as CRM, billing, e-commerce, warehouse, website builder, inventory management, and accounting. The software is distributed in an open-code model, providing a free Community Edition, released under the GNU LGPLv3 software license. The source code of the Community Edition is available on the companies GitHub. Besides the free and open-source Community Edition, a proprietary "Enterprise" version provides additional features and services.
```
Comment (nedaehm) [inline tutorials/odoo/index.mdx:7]:
```suggestion
  Odoo is a business management software, including different modules such as CRM, billing, e-commerce, warehouse, website builder, inventory management, and accounting. The software is distributed in an open-code model, providing a free Community Edition, released under the GNU LGPLv3 software license. The source code of the Community Edition is available on the company's GitHub. Besides the free and open-source Community Edition, a proprietary "Enterprise" version provides additional features and services. 
```

File: tutorials/odoo/index.mdx [hunk 17]
```diff
@@ -159,41 +173,48 @@ This tutorial will show you how to deploy Odoo 14 on a [Scaleway Dedibox](https:
...
-  Once all data is filled in, click **Create database** to create the Odoo database and your first login.
-
-2. The application dashboard displays:
+  Once all data is filled in, click **Create database** to create the Odoo database and your first login. The application dashboard displays:
 
-<Lightbox src="scaleway-odoo_app_dashboard.png" width="900" alt="" />
+  <Lightbox src="scaleway-odoo_app_dashboard.png" width="900" alt="" />
 
-From this panel, you can install additional applications and modules to your Odoo installation. Click the **Install** button next to each app to install it automatically on your Odoo server. You can see more details about each application by clicking on the **Learn More** button.
+3. (Optional): Click the **Install** button next to each app you want to install it automatically on your Odoo server. You can see more details about each application by clicking on the **Learn More** button.
```
Comment (nedaehm) [inline tutorials/odoo/index.mdx:180]:
```suggestion
3. (Optional): Click the **Install** button next to each app you want to install automatically on your Odoo server. You can see more details about each application by clicking on the **Learn More** button.
```

File: tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx [hunk 18]
```diff
@@ -47,7 +47,8 @@ In this example we will use [Scaleway Database for PostgreSQL](https://www.scale
 
 <Message type="requirement">
 
-- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)- You have a [managed Database for PostgreSQL](https://www.scaleway.com/en/database/)
+- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+- You have a [managed Database for PostgreSQL](https://www.scaleway.com/en/database/)
 - You have a compute instance running on Ubuntu Bionic Beaver (18.04) for hosting the Grafana interface
```
Comment (nedaehm) [inline tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx:52]:
```suggestion
- You have a Compute Instance running on Ubuntu Bionic Beaver (18.04) for hosting the Grafana interface
```

File: tutorials/erpnext/index.mdx (hunk 11 above)
Comment (RoRoJ) [reply tutorials/erpnext/index.mdx:295]: I don't think the formatting here changes anything one way or the other? :-)

File: tutorials/erpnext/index.mdx (hunk 12 above)
Comment (RoRoJ) [reply tutorials/erpnext/index.mdx:296]: I don't think the formatting here changes anything one way or the other

**Review by bene2k1 — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

**Pull request merged**